### ISSUE TITLE
[a11y] 11.2.5.1 Pointer gestures

### ISSUE THREAD

**Issue by tschiebel** [owncloud/android#4370]
audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf
https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de

### Image Viewer

- [x] The carousel elements (images) can only be moved using swipe gestures. There is no alternative input option using simple pointer gestures.

**Comment by jesmrec** [comment]
Add visual controls to move between images as an option.

**Comment by JuancaG05** [comment]
Documentation about support for keyboard commands: https://developer.android.com/develop/ui/views/touch-and-input/keyboard-input/commands

**Comment by JuancaG05** [comment]
Support for direction arrow keys is already given in image preview: when pressing `->`, it moves to the next image preview, and when pressing `<-` it moves to the previous one
